roles/common/tasks/system-file-permissions.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Set permissions for system log directory recurisively
+  file: path=/var/log state=directory recurse=yes mode=0640
```

Comment:
i see what jlk is saying now. ill follow up later, ill have to research this more